README.md

```diff
@@ -33,11 +33,14 @@ After the database is initialized, run unit tests:
 ./gradlew test
 ```
 
+To set up serrvice account credentials and other configuration for running locally:
+* Install yq `brew install yq`
```

Comment:
`yq` provides basically the same functionality as `jq`, but for yaml instead of json. 

Since the vault file names don't line up with the property names expected by the service, we need to do some manipulation of the yaml. `yq` seemed the most lightweight way of doing that.